apps/finance/app/index.html

```diff
@@ -5,6 +5,15 @@
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <meta name="theme-color" content="#000000">
     <title>Finance</title>
+    <style>
```

Comment:
Not sure this is the way we'd like to solve this. We should probably find a more generic solution to this with @bpierre.

Reply:
I'm open to suggestions.

Reply:
cc @chadoh maybe he has 2 satoshis to share about this

Reply:
Yes, we should try to avoid requiring boilerplate CSS to make aragonUI components work, if possible.

A solution might be to use [`BaseStyles`](https://github.com/aragon/aragon-ui/blob/newstyle/src/components/BaseStyles/BaseStyles.js) for this, and use `Layout` for the `height: 100%` + `overflow: auto` (the same way `AppView` was doing it).

The idea with Layout is that it can be nested to set another width (to simulate the behavior of iframe apps on internal apps or global settings), so we should ensure it works properly in these cases. We should also make sure this works well with the client itself.

Another solution I can see would be to apply `overflow: hidden` to `html` and `body` when the SidePanel starts its opening transition, but it would create a jump :-/

Reply:
> Another solution I can see would be to apply overflow: hidden to html and body when the SidePanel starts its opening transition, but it would create a jump :-/

Yeah, I want to avoid that as it is the `JS` solution and I think this can be solved by setting the right css rules.

Reply:
The `html` and `body` overflow hidden is more of a safeguard as `#root` should not take more than `100vh`, what I'm thinking is should/could we provide a `main` container for apps for this or should app developers be responsible for it?

Reply:
I think we should provide it yes :-) If we want to avoid having another component, `Layout` is seems the best place to do that? We could add in the documentation that if the layout is disabled (on `Main`), then it is the author’s responsibility to do it.